src/api/module/VBClubMgmt/src/V1/Rest/Api/EventEntityInterface.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace VBClubMgmt\V1\Rest\Api;
+
+interface EventEntityInterface
+{
+    const ID = 'event_id';
+    const NAME = 'name';
+    const DATE = 'date';
+    const DESCRIPTION = 'description';
+    const PARTICIPANTS = 'participants';
+    const DATA = 'data';
+    const CREATED_AT = 'created_at';
+    const UPDATED_AT = 'updated_at';
+
+    /***
+     * @return int
+     */
+    public function getEventId(): int;
```

Comment:
If the project is in PHP7.1, the documentation above this method is a bit useless, no ?

Reply:
Yes. It's juste there for not being considerated as an omission.